packages/aa/src/index.js

```diff
@@ -2,7 +2,7 @@
 
 const { readFileSync, realpathSync, lstatSync } = require('node:fs')
 const path = require('node:path')
-const nodeResolve = require('resolve')
+const nodeResolve = require('resolve/sync')
```

Comment:
it bears repeating that `resolve` does not understand the `exports` field in `package.json` and will likely cause problems soon.  I understand we use it to _avoid_ reading `package.json`, but there's really no way to avoid it now. 

¯\\_(ツ)_/¯

Reply:
Note that this doesn't rely on exports in `package.json` (in fact, [`resolve`](https://www.npmjs.com/package/resolve) has none) but requires the module by subpath, which should not be problematic?